app/build.gradle

```diff
@@ -19,6 +20,10 @@ android {
 }
 
 dependencies {
+
+    compile 'com.android.support:appcompat-v7:23.2.0'
```

Comment:
we support from version 27, do we need this line ?

Reply:
it can be deleted you're right, I forgot to delete